### ISSUE TITLE
dead link in help docs

### ISSUE THREAD

**Issue by hobrien** [hartleys/JunctionSeq#12]
Hi,

Thanks for writing this package. It looks like exactly what we need to deal with novel splice sites in our DEXseq analyses. I've encountered a couple of extremely minor issues that I thought I'd bring to your attention (I'll post them separately). The first concerns the help documentation for plotJunctionSeqResultsForGene. Under colorList it says:

```
A named list of R colors, setting the colors used for various things. See link{junctionSeqColors} for more information.
```

but the link isn't working

**Comment by hartleys** [comment]
Just a typo. It will be fixed in the next release (which I'll post once I fix the other problems you've found).

**Comment by hartleys** [comment]
Fixed in the new release (v1.3.5)
